pkg/controller/stacks/install/stackinstall.go

```diff
@@ -273,6 +315,18 @@ func (h *stackInstallHandler) delete(ctx context.Context) (reconcile.Result, err
 		return fail(ctx, h.kube, h.ext, err)
 	}
 
+	stackControllerNamespace := h.ext.GetNamespace()
+	if h.hostAwareConfig != nil {
+		stackControllerNamespace = h.hostAwareConfig.HostControllerNamespace
+	}
+
+	// Once the Stacks are gone, we can remove install job associated with the StackInstall using hostKube since jobs
+	// were deployed into host Kubernetes cluster.
+	if err := h.hostKube.DeleteAllOf(ctx, &batchv1.Job{}, client.MatchingLabels(labels),
```

Comment:
do we need to delete the deployments of the stack controllers as well? (or remind me on how they get cleaned up again)

EDIT: nevermind, found it below in stack.go's delete func :)